app/service/account.py

```diff
@@ -28,3 +39,17 @@ async def read_accounts(user: dict = Depends(get_current_user)) -> List[Account]
 async def read_account(account_id: int, user: dict = Depends(get_current_user)) -> Account:
     account = session.query(Account).filter(Account.id == account_id).first()
     return account
+
+
+async def update_account(account: Optional[Account], user: dict = Depends(get_current_user)) -> Account:
+    new_account = session.query(Account).filter(Account.id == account.id).first()
+    if account is None or new_account is None:
+        return account_not_found()
```

Comment:
P2: account is None은 인자로 받은 account가 없는 경우인데, 클라이언트에서 업데이트에 사용할 가계부를 서버에게 전달했을 때, 그 가계부에 대한 포맷 검증은 API쪽에서하고, Service에서는 업데이트만 담당하는 것이 좋아보입니다.
그리고 다른 예외들은 raise 키워드로 예외를 던지고 있는데 여기서는 return을 하고있습니다. raise로 HTTPException을 던져야하지않나 싶습니다. 확인부탁드릴께요.
```suggestion
    if new_account is None:
        return account_not_found()
```